config/tests/jobs/jobs_test.go

```diff
@@ -331,6 +331,11 @@ func TestTrustedJobs(t *testing.T) {
 			if err := validateImagePushingImage(post.Spec); err != nil {
 				t.Errorf("%s defined in %s %s", post.Name, post.SourcePath, err)
 			}
+			// jobs in image-pushing have a wider approval pool than test-infra-oncall
+			// and so shouldn't be alllowed to run in test-infra-trusted cluster
+			if post.Cluster != wgK8sTrusted {
```

Comment:
a) please move references to `wgK8sTrusted` and `image-pushing` out of `TestTrustedJobs`
b) Create a `TestImagePushingJobs` that validates that `image-pushing` jobs always use the `wgK8sTrusted` cluster.

Reply:
I opted to make a `TestK8sInfraTrusted`, as the cluster is intended to run more than jobs in image-pushing/, and I'm inclined to leave image-pushing/ where it is for discoverability rather than pull everything under one trusted root